src/gl/lineFeature.js

```diff
@@ -60,8 +95,15 @@ var gl_lineFeature = function (arg) {
           'uniform mat4 projectionMatrix;',
           'uniform float pixelWidth;',
           'uniform float aspect;',
+          'uniform float miterLimit;',
+          'uniform float antialiasing;',
 
           'varying vec4 strokeColorVar;',
+          'varying vec4 subpos;',  /* px, py, length - px, width */
+          'varying vec4 info;',  /* near mode, far mode, offset */
+          'varying vec4 angles;', /* near angle cos, sin, far angle cos, sin */
+
+          'const float PI = 3.14159265358979323846264;',
```

Comment:
Let's do that when we address issue #655 (in a different PR).

Alternately, we could use the value in javascript, i. e., `'const float PI = ' + Math.PI.toFixed(20) + ';'`.  We aren't guaranteed the same precision choices in javascript and WebGL, though.  The differences almost certainly don't matter much here.